build.gradle

```diff
@@ -111,6 +117,35 @@ def getVersionCodeFromManifest() {
     return Integer.parseInt(matcher.group(1))
 }
 
+task downloadNative(type: Copy) {
+    def release = new JsonSlurper().parseText("https://api.github.com/repos/syncthing/syncthing/releases/latest".toURL().text)
+
+    // Download ARM
+    new File('bin').mkdirs()
+    copy {
+        ant.get(src: "https://github.com/syncthing/syncthing/releases/download/v0.10.30/syncthing-linux-arm-"+release.tag_name+".tar.gz", dest: "bin/syncthing-arm.tar.gz")
```

Comment:
This doesn't make sense, you're hardcoding the version in one place and reading it from the API in another?